src/install/mongodb/intro.mdx

```diff
@@ -0,0 +1,22 @@
+---
+headingText: Before we start
+componentType: default
+---
+
+import infrastructureMongoDashboardCropped from 'images/infrastructure_screenshot-crop_mongodb-dashboard-cropped.webp'
+
+Our MongoDB integration collects and sends metrics from your MongoDB database to our platform, where you can monitor throughput, collection size, connections, indexes, and more
+
+With this data, you can use New Relic to create alerts when throughput spikes, create dashboards to monitor collection size, and more.
```

Comment:
```suggestion
Gain deep insights into MongoDB performance with seamless data integration into New Relic. Monitor key metrics like throughput, collection size, connections, and indexes effortlessly, all within our powerful platform. Create alerts to stay ahead of spikes, build custom dashboards for tailored views, and proactively optimize your database monitoring.
```

Consider rewriting this intro (something like this ^) to avoid having two ", and more..." sentences.